resource-managers/mesos/src/main/scala/org/apache/spark/scheduler/cluster/mesos/MesosCredentialRenewer.scala

```diff
@@ -63,7 +63,8 @@ class MesosCredentialRenewer(
 
   def scheduleTokenRenewal(): Unit = {
     def scheduleRenewal(runnable: Runnable): Unit = {
-      val remainingTime = timeOfNextRenewal - System.currentTimeMillis()
+      // val remainingTime = timeOfNextRenewal - System.currentTimeMillis()
+      val remainingTime = 5000
```

Comment:
Why 5000?

Reply:
well that's embarrassing, just a debugging tool that I forgot to remove.